src/main/java/Assistinator/Command.java

```diff
@@ -0,0 +1,14 @@
+package Assistinator;
+
+public enum Command {
+    BYE, LIST, MARK, UNMARK, TODO, EVENT, DEADLINE, DELETE, FIND, UNKNOWN;
+
+    static Command fromString(String input) {
```

Comment:
Might consider naming the method `parse` or `parseCommand` something similar instead of fromString to be more descriptive